examples/rtl_433_mqtt_hass.py

```diff
@@ -158,7 +158,15 @@
             "state_class": "measurement"
         }
     },
-
+    "time": {
```

Comment:
It seems like there was a lot of thought into what to do and why but it is not explained in comments.

Reply:
I had left notes about this at https://github.com/merbanan/rtl_433/pull/1665#discussion_r799932852

However, I did a deeper look, and I am back on the side of "this isn't needed". Getting the "Last updated" value in home assistant requires setting `force_update` in the MQTT configuration: https://www.home-assistant.io/integrations/sensor.mqtt/#force_update

I've filed a docs PR as currently "updated" doesn't match anything to this: https://github.com/home-assistant/home-assistant.io/pull/23900

The only reason I can think to keep this is if somehow there are sensors out there that don't provide _any_ values  when transmitting to associate a "Last changed" value with?

I'm going to post in the discourse thread to ask for feedback on removing this.

Reply:
I ended up posting a home assistant docs PR, and in summary I misunderstood what "Last updated" referred to: https://github.com/home-assistant/home-assistant.io/pull/23900

Instead, we should set a timestamp as a diagnostic entity, and possibly have them disabled by default like zigbee2mqtt does: https://developers.home-assistant.io/blog/2021/10/26/config-entity/

Reply:
This sensor is now marked as a diagnostic sensor and disabled by default. This matches what zigbee2mqtt does.